### ISSUE TITLE
PPA package installs without ZeroMQ enabled

### ISSUE THREAD

**Issue by tomx-eth** [gandrewstone/bitcoin#1302]
After switching from Bitcoin ABC to Bitcoin Unlimited, I get this error when starting bitcoind.

`Error reading configuration file: unrecognized option 'zmqpubrawtx'`

Which leads me to believe that it's failing to read my zeromq settings from bitcoin.conf because it's not enabled. Docs say that it should be enabled by default.

**Comment by ptschip** [comment]
are you running a windows version of BU?

**Comment by tomx-eth** [comment]
Ubuntu 16.04.4 x64

**Comment by sickpig** [comment]
@starvin thanks for the report. ~going to fix it and rebuild the ppa packages~

How did you install BU on your system? Did you download directly from the BU site or did you used the Ubuntu PPA repository?

In any case could you check if you you have `libzmq3` installed in your system? Just run `dpkg -l | grep libzmq3` and paste the output here.

**Comment by sickpig** [comment]
@starvin 

find the issue, libzmq3 was removed by the list of dependencies by chance (only for xenial). I'm currently rebuilding the package, going to ping you as soon as I got it done and published.

**Comment by tomx-eth** [comment]
@sickpig 

Thanks! I installed from `ppa:bitcoin-unlimited/bucash`, but I don't have libzmq3 installed. Looks like you found the issue.

However, libzmq3 also wasn't installed with ABC, so is this unique to BU? Looks like they both include `libzmq5`.

**Comment by sickpig** [comment]
BU and ABC both include libzmq3 as min dependency. Both of them specify `libzmq3-dev` in their dependency list, which implies that libzmq3 will also be installed. That said I guess that zmq5 should work the same as long it is backward compatible with v3.

The problem is that for some reason I dissed zmq3 from xenial during the updating of the repo to 1.4.0.0. I tried to rebuild after re-adding libzmq3 dependency but for some reason it fails. 

Looking at the problem now, will let you now once I got it fixed.I mean manually installing zmq will fix the problem but the actual fix is getting the `apt` to install zmq lib automatically like it did before.
